common/domain/replicationTaskExecutor_test.go

```diff
@@ -248,3 +249,195 @@ func TestDomainReplicationTaskExecutor_Execute(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleDomainCreationReplicationTask(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockDomainManager := persistence.NewMockDomainManager(ctrl)
```

Comment:
so, the problem with putting it up here is that you're sharing it between table-rows, so it's a little tricky to get the assertions